documentation/modules/cruise-control/proc-generating-optimization-proposals.adoc

```diff
@@ -125,7 +125,23 @@ Status:
 ----
 +
 The properties in the `Optimization Result` section describe the pending cluster rebalance operation. 
-For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals]. 
+For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals].
+
+.Insufficient CPU capacity
+
+If the Kafka cluster is overloaded in terms of CPU utilization and needs some additional brokers, you may see an error like the following in the `KafkaRebalance` status. This may happen because, while proposals exclude the `excludedTopics`, the CPU utilization still considers replicas of the `excludedTopics`.
```

Comment:
The paragraph describes under which conditions you may see this error, so that you can verify if it matches with your setup. Instead, the note specifically addresses any doubts you may have looking at numbers. I would prefer the note to focus only on that aspect, as we know that it can be misleading.

Reply:
I felt that the second sentence starting with "This may happen because" makes it sound as if `excludedTopics` could be the cause of the brokers being overloaded. From what I understand, `excludedTopics` does not affect the load on the brokers, it affects what partitions are allowed to be moved?

I thought the mention of `excludeTopics` was to warn users who are expecting `excludeTopics` to affect the load that `excludeTopics` has no affect on the load. Was that the intention or am I misunderstanding something?

Reply:
I am fine with leaving the mention of `excludeTopics` up here in the first paragraph to prevent user confusion but I think we should rephrase so something like: 

> It's worth noting that this utilization value is unaffected by the excluded topics configuration (`excludedTopics`). Although optimization proposals will not reassign replicas of excluded topics, their load is still considered in the utilization calculation.

Also we will want one sentence per line in this file as well!